### ISSUE TITLE
Google play game services plugin class duplication with support annotations v23.1.1

### ISSUE THREAD

**Issue by TheSubhajitK** [playgameservices/play-games-plugin-for-unity#965]
Hi,
My project has In-App purchases, Push Notification, Facebook SDK, MMedia Ads. I tried to integrate the play-games for unity plugin while developing my project in Unity 4.6. But, the authentication won't work, so I followed the guide advice in the Git repo and upgraded my project to Unity 5. After removing the google play games services plugin, i wan able to make a build. The Authentication works on Google Play, but, my push notification part doesn't as I removed the above-mentioned plugin. I would be very obliged, if you could provide me some help with this, about, how do i get my Push Notifications as well, as your plugin work at the same time.

**Comment by claywilkinson** [comment]
When you say it does not work, do you mean at runtime, or is it a compile time issue?  Can you share a log of the problem?

**Comment by TheSubhajitK** [comment]
Well, it was a compile time issue, when I used the google play services jar file, which produced a top level Exception and Unity was unable to convert the project into Dex format. I searched a lot about how to overcome this problem, finally, understood that there were a lot of classes in the google play services jar, which conflicted with your plugin's jar files. Opened up the google play services jar file using WinRar and removed those classes, one by one, added the GCM folder containing the Google Cloud Messaging classes, and voila, my push notifications worked, authentication happened. It was great. So, I finally solved my own problem. Thanks, for making this plugin,by the way.
